app/src/main/java/com/example/keniphhomemaintenance/dwellings/NewDwellingDialog.kt

```diff
@@ -0,0 +1,81 @@
+package com.example.keniphhomemaintenance.dwellings
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.Button
+import androidx.compose.material.Divider
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.example.keniphhomemaintenance.R
+import com.example.keniphhomemaintenance.ui.cookbook.KeniphOutlinedTextField
+
+@Composable
+fun NewDwellingDialog(
+    visibilityState: MutableState<Boolean>,
+    modifier: Modifier = Modifier,
+    newName: MutableState<String> = remember { mutableStateOf("") },
```

Comment:
Why are these parameters?

Reply:
- visibilityState toggles whether the dialog is visible or invisible, and at the end of the dialog you have to turn it off
- the rest of the parameters were removed